app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class RentsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
This is the second controller we're using it in, what do you think about moving it to the `ApplicationController`? Or we can make a specific `ApiController` which all endpoints that expose an API can inherit from, and add it there

Reply:
I do not know how the training continues, so I limit myself to doing these actions as I face the cards. Now, your idea with respect to an `ApiController` seems good to me, it's a way to control those cases.